### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by nati2323**
- Added Media Query in my CSS file.
- Added Grid and Flexbox to lay out the elements on the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -344,17 +344,14 @@ <h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
...
-              <label for="email" class="fw-500 fz-17 lh-20">Email:</label>
-              <input type="email" id="email" class="contact__email fw-500 fz-17 lh-20" placeholder="Enter your email"
+               <input type="email" id="email" class="contact__email fw-500 fz-17 lh-20" placeholder="Enter your email"
                 name="contact_email" required/>
             </li>
             <li>
-              <label for="message" class="fw-500 fz-17 lh-20">Message:</label>
               <textarea id="message" class="fw-500 fz-17 lh-20" name="contact_message" placeholder="Write your message here"
                 cols="30" rows="8" maxlength="500" required></textarea>
             </li>
```
Comment (sja-thedude) [inline index.html]: Mobile version:

- [ ] Kindly add the background image for the section as per the Figma design shown.
- [ ] Kindly make sure to change "My Projects" to "LET'S CONNECT" as per the Figma design.

https://user-images.githubusercontent.com/70269507/166904386-1768cafc-d765-4d70-a5c4-c103d9224b19.mov

Comment (sja-thedude) [inline index.html]: Mobile Version:
- [ ] Kindly add the background image for the contact section as per the Figma design.
- [ ] Kindly change the placeholder for the input name and email field as per the Figma design.
- [ ] Kindly follow the text colour for the placeholder message text.
- [ ] Kindly follow the font weight for the paragraph as per the Figma design. 

https://user-images.githubusercontent.com/70269507/166905256-89a9d8bf-8f83-4b85-acfa-f8773a92fc07.mov

Comment (sja-thedude) [inline index.html]: Mobile version:

- [ ] Kindly follow the correct font weight for the paragraph as per the Figma design.

https://user-images.githubusercontent.com/70269507/166906375-5513de21-1401-4712-97e9-afb5c5ba307e.mov

Comment (sja-thedude) [inline index.html]: Mobile version:

- [ ] Kindly use the same images as given in the Figma design for the mobile version.
- [ ] Kindly Follow the same font-weight for the paragraph as per the Figma design.

https://user-images.githubusercontent.com/70269507/166907037-129dc94a-960e-4a11-b9f6-769502077101.mov

- [ ] Kindly follow the same text as per the given Figma design as given in the video below.
- [ ] Kindly follow the colours and background colour for lists as shown below.

https://user-images.githubusercontent.com/70269507/166908082-7799a246-1f2b-440b-8d76-3824630861ee.mov

Comment (sja-thedude) [inline index.html]: Desktop Version:

- [ ] Kindly make sure you are following the Figma design for lists and displaying as shown for all the cards. 

https://user-images.githubusercontent.com/70269507/166908817-6cac64f0-0139-42d6-a464-fcaba0fe52d7.mov

File: index.html [hunk 2]
```diff
@@ -344,17 +344,14 @@ <h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
...
           <ul class="ul">
             <li>
-              <label for="fname" class="fw-500 fz-17 lh-20">Name:</label>
               <input type="text" id="fname" class="contact__name fw-500 fz-17 lh-20" name="contact_name"
                 placeholder="Enter your name" required maxlength="30" />
             </li>
             <li>
-              <label for="email" class="fw-500 fz-17 lh-20">Email:</label>
-              <input type="email" id="email" class="contact__email fw-500 fz-17 lh-20" placeholder="Enter your email"
+               <input type="email" id="email" class="contact__email fw-500 fz-17 lh-20" placeholder="Enter your email"
```
Comment (sja-thedude) [inline index.html]: Mobile version:

- [ ] Kindly follow the width for the input fields and the message field as per the Figma design. Also make sure, it is not round at the edges.

- Figma design:

Screenshot 2022-05-05 at 1 53 08 PM

- Your design:

Screenshot 2022-05-05 at 1 53 26 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @nati2323, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ All linters passing
✔️ Good PR description and title
✔️ Correct Gitflow

## Required 
- [ ]  Kindly make sure to use both Flexbox and Grid in order to layout the elements on the page for the desktop version also. You have implemented flexbox and grid in the mobile version. Kindly implement it for the desktop version.
- [ ] Kindly update your readme file with a correct description of the project and with your details.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @nati2323 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has one of the linter checks failing kindly fix it.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html (hunk 1 above)
Comment (danmainah) [reply index.html]: - [ ] Kindly implement the changes requested by the reviewer on the form, your form doesn't have background images as is in the design.

Comment (danmainah) [reply index.html:357]: - [ ] Kindly implement the changes requested by the reviewer, the reviewer has a video showing the comparison of your design and the Figma design given.

Comment (danmainah) [reply index.html]: - [ ] Kindly implement the changes requested by the reviewer, the reviewer has a video showing the comparison of your design and the Figma design given.

Comment (danmainah) [reply index.html]: - [ ] Kindly implement the changes requested by the reviewer, the reviewer has a video showing the comparison of your design and the Figma design given.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @nati2323, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️

- [ ] Kindly implement the changes requested by the reviewer, the reviewer has videos showing the comparison of your design and the Figma design given in different sections where you made an error.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nati2323** [comment]
- Sorry for the Linters error now I have fixed all of them.

**Review by prolajumokeoni — APPROVED** [review]
Hi @nati2323, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nati2323** [comment]
***@***.***

**Comment by nati2323** [comment]
***@***.***,

**Comment by nati2323** [comment]
***@***.***

**Pull request merged**